src/imgui/ImGuiUtils.cc

```diff
@@ -283,6 +284,160 @@ std::string getShortCutForCommand(const HotKey& hotkey, std::string_view command
+        case ImGuiKey_I: return SDLK_i;
+        case ImGuiKey_J: return SDLK_j;
+        case ImGuiKey_K: return SDLK_k;
+        case ImGuiKey_L: return SDLK_l;
+        case ImGuiKey_M: return SDLK_m;
+        case ImGuiKey_N: return SDLK_n;
+        case ImGuiKey_O: return SDLK_o;
+        case ImGuiKey_P: return SDLK_p;
+        case ImGuiKey_Q: return SDLK_q;
+        case ImGuiKey_R: return SDLK_r;
+        case ImGuiKey_S: return SDLK_s;
+        case ImGuiKey_T: return SDLK_t;
+        case ImGuiKey_U: return SDLK_u;
+        case ImGuiKey_V: return SDLK_v;
+        case ImGuiKey_W: return SDLK_w;
+        case ImGuiKey_X: return SDLK_x;
+        case ImGuiKey_Y: return SDLK_y;
+        case ImGuiKey_Z: return SDLK_z;
+        case ImGuiKey_F1: return SDLK_F1;
+        case ImGuiKey_F2: return SDLK_F2;
+        case ImGuiKey_F3: return SDLK_F3;
+        case ImGuiKey_F4: return SDLK_F4;
+        case ImGuiKey_F5: return SDLK_F5;
+        case ImGuiKey_F6: return SDLK_F6;
+        case ImGuiKey_F7: return SDLK_F7;
+        case ImGuiKey_F8: return SDLK_F8;
+        case ImGuiKey_F9: return SDLK_F9;
+        case ImGuiKey_F10: return SDLK_F10;
+        case ImGuiKey_F11: return SDLK_F11;
+        case ImGuiKey_F12: return SDLK_F12;
+        case ImGuiKey_F13: return SDLK_F13;
+        case ImGuiKey_F14: return SDLK_F14;
+        case ImGuiKey_F15: return SDLK_F15;
+        case ImGuiKey_F16: return SDLK_F16;
+        case ImGuiKey_F17: return SDLK_F17;
+        case ImGuiKey_F18: return SDLK_F18;
+        case ImGuiKey_F19: return SDLK_F19;
+        case ImGuiKey_F20: return SDLK_F20;
+        case ImGuiKey_F21: return SDLK_F21;
+        case ImGuiKey_F22: return SDLK_F22;
+        case ImGuiKey_F23: return SDLK_F23;
+        case ImGuiKey_F24: return SDLK_F24;
+        case ImGuiKey_AppBack: return SDLK_AC_BACK;
+        case ImGuiKey_AppForward: return SDLK_AC_FORWARD;
+	default: return SDLK_UNKNOWN;
+    }
+    return ImGuiKey_None;
+}
+
+std::optional<std::string> getKeyChordName(ImGuiKeyChord keyChord)
+{
+	int keyCode = ImGuiKeyToSDL2_KeyCode(ImGuiKey(keyChord & ~ImGuiMod_Mask_));
+	if (keyCode != SDLK_UNKNOWN) {
+		auto scanCode = SDL_GetScancodeFromKey(keyCode);
+		return strCat((keyChord & ImGuiMod_Ctrl ? "Ctrl+" : ""),
+			(keyChord & ImGuiMod_Shift ? "Shift+" : ""),
+			(keyChord & ImGuiMod_Alt ? "Alt+" : ""),
+			(keyChord & ImGuiMod_Super ? (ImGui::GetIO().ConfigMacOSXBehaviors ? "Cmd+" : "Super+") : ""),
+			SDL_GetScancodeName(scanCode));
+	}
+	return {};
+}
+
+std::optional<ImGuiKeyChord> getKeyChordValue(const std::string_view name)
+{
+	ImGuiKeyChord keyMods = {};
+	keyMods |= StringOp::containsCaseInsensitive(name, "Ctrl+")  ? ImGuiMod_Ctrl  : 0;
+	keyMods |= StringOp::containsCaseInsensitive(name, "Shift+") ? ImGuiMod_Shift : 0;
+	keyMods |= StringOp::containsCaseInsensitive(name, "Alt+")   ? ImGuiMod_Alt   : 0;
+	SDL_Scancode scanCode = SDL_GetScancodeFromName(std::string(name).c_str());
```

Comment:
Is there a specific reason why you choose `SDL_getScancodeFromName()` instead of `SDL_GetKeyFromName()`?
I think there is a significant difference between both:
* scancodes are keyboard-layout agnostics. E.g. `SDL_SCANCODE_Q` always refers to the top-left letter-key, on a QWERTY-layout that is the 'Q' key, but on an AZERTY-layout this will be the 'A' key.
* keycodes do take the keyboard-layout into account. E.g. `SDLK_A` always refers to the key with the label 'A', even if that key is in different positions on a QWERTY vs AZERTY layout.

I think for shortcuts it's better to use keycodes, no?